### DISCUSSION TITLE
How to customize a page's SEO properties if the <head> exists in a base layout

### DISCUSSION THREAD

**Discussion by sean-au** [jonasmerlin/astro-seo#54] [Q&A]
How to customize a page's SEO properties if the <head> exists in a base layout?

Say <head> lives in BaseLayout, and I have set up all the defaults there...

and for a page I do:

```
---
import BaseLayout from '../layouts/BaseLayout.astro;
---
<BaseLayout >


</BaseLayout>
```

Is it possible to define additional config or overrides at this page level?

**Comment by jonpage0** [comment]
I believe the answer would be to export the props you want to customize in the BaseLayout.astro component and then reference them when using <SEO />.

**Comment by orangecoloured** [comment]
@sean-au How did you solve this? I've started with Astro and having hard time overriding the meta in the nested layouts.

**Comment by sean-au** [comment]
Unfortunately I did not work this out. I continued with HUGO and a SEO
plugin that I wrote for Hugo. If you need more info let me know. It's very
different to Astro. Hard to compare.

**Reply by simpson** [reply]
I'm starting to really wish I used Hugo....

**Comment by simpson** [comment]
Is this dead?
